test/functional/spawn.js

```diff
@@ -0,0 +1,18 @@
+var execa     = require('execa');
+var Promise   = require('pinkie');
+var assign    = require('lodash').assign;
+
+
+module.exports = function (cmd, args, opts) {
+    opts = opts || {};
```

Comment:
Nevermind, actually there is no need for it here, I've figured how to use `execa`. BTW it's here just for debugging purposes, it will be removed just after I will be done with Firefox on AppVeyor.